src/app/auth/authentication.service.ts

```diff
@@ -145,6 +151,108 @@ export class AuthenticationService {
     localStorage.setItem('refresh_token', token.refresh_token);
     return token;
   }
+  
+  connectGitHub(redirectUrl: string) {
+    //let headers = new Headers({ 'Content-Type': 'application/json' });
+    //let options: RequestOptions = new RequestOptions({ headers: headers });
+    let tokenUrl = this.apiUrl + 'token/link';
+    let form = document.createElement("form");
+
+    // Create the for parameter
+    let inputFor = document.createElement("input");
+    inputFor.type = "hidden";
+    inputFor.name = "for";
+    inputFor.value = "https://github.com";
+    form.appendChild(inputFor);
+
+    // Create the token parameter
+    let inputToken = document.createElement("input");
```

Comment:
There shouldn't be any html code in here - maybe we should pair on this after the holiday and discuss a standard approach.  The login-client library should be logic only, all presentation code (HTML, inputs) should be in the fabric8-ui Angular code.